DataRepo/views/models/infusate.py

```diff
@@ -23,3 +23,23 @@ def get_context_data(self, **kwargs):
         context["tracer_df"] = tracer_data
 
         return context
+
+
+class InfusateListView(ListView):
+    """Generic class-based view for a list of infusates"""
+
+    model = Infusate
+    context_object_name = "infusate_list"
+    template_name = "DataRepo/infusate_list.html"
+    ordering = ["name"]
+
+    def get_context_data(self, **kwargs):
+        # Call the base implementation first to get the context
+        context = super(InfusateListView, self).get_context_data(**kwargs)
```

Comment:
NB. I think this usage is deprecated.
```suggestion
        context = super().get_context_data(**kwargs)
```

Reply:
fixed!